templates/keras_cv/bounding_box/index.md

```diff
@@ -0,0 +1,8 @@
+# KerasCV Bounding Boxes
+All KerasCV components that process bounding boxes require a `bounding_box_format`
+argument.  This argument allows you to seamlessly integrate KerasCV components into
+your own workflows while preserving proper behavior of the components themselves.
+
+[The list of bounding box formats supported in KerasCV are listed in the API docs](/api/keras_cv/bounding_box/formats).
```

Comment:
`The bounding box formats supported in KerasCV [are listed in the API docs](/api/keras_cv/bounding_box/formats)`